stdlib/public/core/Array.swift

```diff
@@ -1798,7 +1798,7 @@ extension Array {
   ///             destBytes.copyBytes(from: srcBytes)
   ///         }
   ///     }
-  ///     // numbers == [1, 2]
+  ///     // numbers == [1, 2] on little-endian machine. https://en.wikipedia.org/wiki/Endianness
```

Comment:
This is better, but I think it could be still more improved if (a) this is an actual link; and (b) it refers to a source that is more vetted--since the Wikipedia article is tagged "This article has multiple issues" (which is not a good look for a reference)--and more concise--since the Wikipedia article is also many times longer than this documentation and contains more than any user would need to look up to understand this documentation.

Before we start, there is one other problem to fix along the way: Someone else fixed the example so that `numbers` is of type `[Int32]` instead of `[Int]` so that word size doesn't matter. This is very good for didactic purposes; we do not need to teach someone about word size in this place, so let's not do so. However, they forgot to fix the text that preceded it, so it still says "`numbers`, an array of `Int`"--this is incorrect and needs to be fixed to say "an array of `Int32`".

Here is what I would suggest: Remove the clarifying information here from this line and introduce it in the text that introduces the example like this:

> The following example copies bytes from the `byteValues` array into `numbers`, an array of `Int32`; on a [little-endian](#) machine, the result is as shown:

Now, what resource might be a good one for defining endianness? A quick Google search shows a very [readable and accurate definition at MDN](https://developer.mozilla.org/en-US/docs/Glossary/Endianness). It covers essentially everything a reader of this documentation might need to know, and crucially, very little that a reader _wouldn't_ need to know. I like it, but perhaps you might prefer something similar from a different source; just make sure that it is concise, written at an appropriate level, and vetted.

Reply:
> (a) this is an actual link

should it be `<a href="https://developer.mozilla.org/en-US/docs/Glossary/Endianness/">little-endian</a>` or `[little-endian](https://developer.mozilla.org/en-US/docs/Glossary/Endianness/)`?